packages/components/src/interfaces/native/Prose.ts

```diff
@@ -10,15 +10,17 @@ import type { UnorderedListProps } from "./UnorderedList"
 export type ProseContent = (
   | DividerProps
   | HeadingProps
-  | ImageProps
   | ListItemProps
+  | ImageProps
```

Comment:
@sehyunidaaa - ok if prose can contain image? i saw in figma that prose shouldn't. i'm happy to edit and make this change

Reply:
note: not ok - we need to remove